lib/labeler/labeledgegroup.js

```diff
@@ -18,8 +18,10 @@ export default class LabelEdgeGroup {
   getLabelTextArray () {
     var textArray = []
     forEach(this.renderedSegment.pathSegment.getPatterns(), pattern => {
-      var shortName = pattern.route.route_short_name
-      if (textArray.indexOf(shortName) === -1) textArray.push(shortName)
+      textArray.push(
+        pattern.route.route_short_name ||
+          pattern.route.route_long_name
```

Comment:
What happens if both `route_short_name` and `route_long_name` are null?

Reply:
That would be an invalid GTFS. http://gtfs.org/reference/static/#routestxt